modules/ROOT/partials/configuration/extended_valid_elements.adoc

```diff
@@ -7,7 +7,7 @@ When adding a new attribute by specifying an existing element rule (e.g. `+img+`
 
 The below example replaces the current `+img+` rule (including the global element rules).
 
-Type : `+String+`
```

Comment:
This and other changes like this shouldn't have been done, as everywhere now uses this style and it's what Tyler changed it to. I believe it was meant to be definition lists instead (e.g. `::`) but I'm personally not a fan of that for these.

Either way, this is something we need to change en masse if we're going to and shouldn't be done for these couple of specific cases. It probably needs a discussion or chat with Brian to figure out what's best and then should be done via a separate PR.

- [ ] Revert removing these spaces for now

Reply:
Ah, I didn't release that. Good to know. Hopefully should be good now